packages/test-apps/react-vite/package.json

```diff
@@ -8,7 +8,6 @@
     "start": "vite",
     "prebuild": "rimraf dist",
     "build": "tsc && vite build",
-    "build:vercel": "npm run build --prefix ../core && npm run build --prefix ../components && npm run build",
```

Comment:
This doesn't get used anymore. It looks like this was part of the old vercel build config before some stuff moved to github actions